qiskit/quantum_info/operators/measures.py

```diff
@@ -115,7 +115,7 @@ def process_fidelity(channel, target=None, require_cp=True, require_tp=True):
     if isinstance(target, Operator):
         # Compute fidelity with unitary target by applying the inverse
         # to channel and computing fidelity with the identity
-        channel = channel.compose(target.adjoint())
+        channel = channel @ target.adjoint()
```

Comment:
This looks like a behavioural change - `compose` and `dot` aren't the same.  I'm not sure which is the correct way round here (and I'm not keen that we don't have a test failure if the behaviour changes), but the `compose` form feels more correct to me.

Reply:
Thnak you, you're right. I changed it by mistake. reset my change.